### ISSUE TITLE
Time Zone Names

### ISSUE THREAD

**Issue by ClaireBookworm** [nikolaeu/numi#185]
It seems Numi doesn't have standard time zone names or doesn't save locations correctly. Times are a bit arbitrary, or I'm putting in the wrong time. 
![Screen Shot 2020-10-08 at 1 32 24 PM](https://user-images.githubusercontent.com/41759033/95510409-b43cd100-096a-11eb-846e-833b19f61c78.png)

**Comment by mpash** [comment]
Yes it's actually shocking how little continuity there is between dates.

![image](https://user-images.githubusercontent.com/48454220/96811317-7d8e9e00-13e1-11eb-8164-9b24b844563f.png)
